src/compiler/transpile.ts

```diff
@@ -1,14 +1,40 @@
 import { rollupPluginUtils } from '@compiler-deps';
+import {
```

Comment:
Would this also work?
```suggestion
import type {
```

Reply:
I think so!

Reply:
made the change, but I accidentally squashed it into the most recent commit on the branch